lib/internal/Magento/Framework/CompiledInterception/Test/Unit/CompiledInterceptor/_out_interceptors/ComplexItem.txt

```diff
@@ -0,0 +1,193 @@
+namespace Magento\Framework\CompiledInterception\Test\Unit\Custom\Module\Model\ComplexItem;
+
+use Magento\Framework\Config\ScopeInterface;
+use Magento\Framework\ObjectManagerInterface;
+
+/**
+ * Interceptor class for @see \Magento\Framework\CompiledInterception\Test\Unit\Custom\Module\Model\ComplexItem
+ */
+class Interceptor extends \Magento\Framework\CompiledInterception\Test\Unit\Custom\Module\Model\ComplexItem
+{
+    /**
+     * @var ScopeInterface
+     */
+    private $____scope = null;
```

Comment:
As reading parent class fields would add even more to generation time this is to avoid name collision. I know 4 seem paranoid but on the other hand generated code clearly stands out witch is good i suppose. Do we have any guideline/standard for generated code in such case?

Reply:
We have a rule according to PSR-2 (https://www.php-fig.org/psr/psr-2/#42-properties) that properties shouldn't have an underscore in names, I believe, even `__` usage is a very rare case.